### DISCUSSION TITLE
Does new reports inherith new software features?

### DISCUSSION THREAD

**Discussion by matteoperazzolo** [globaleaks/globaleaks-whistleblowing-software#4226] [Q&A] [answered]
### What version of GlobaLeaks are you using?

4.13.11

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Linux

### Describe the issue

We have seen that the ability to archive documents has been added to the platform with the latest updates. I ask you, will this functionality also be activated with old reports or only with those following the update?

**Comment by evilaliv3** [comment, marked as answer]
yes the software is designed to be always retrocompatible and offer new features to old reports.

For example the feature that you are referring is now available on all old reports.
